src/Core/Models/Table/OrganizationUser.cs

```diff
@@ -17,6 +18,15 @@ public class OrganizationUser : ITableObject<Guid>, IExternal
         public string ExternalId { get; set; }
         public DateTime CreationDate { get; internal set; } = DateTime.UtcNow;
         public DateTime RevisionDate { get; internal set; } = DateTime.UtcNow;
+        public bool AccessBusinessPortal { get; set; }
```

Comment:
Question (don't be cross), but to "future-proof" this a bit, like we did with cipher, SSO config, etc. wouldn't it behoove us to create a class to represent these permissions and just store the serialized JSON in the DB in a single column, so the update/save/insert stored procedures never need to change, and the table never needs to change, and we can just add new permissions in the future w/o mucking with the database "stuff"?

Reply:
Implemented!